### ISSUE TITLE
PSCX gcb conflicts with existing gcb on Windows 10

### ISSUE THREAD

**Issue by u0m3** [Pscx/Pscx#14]
Windows 10 seems to come with it's own version of **Get-Clipboard** cmdlet. Is there a way to disable loading it if it already exists? Or should I just overwrite it?
```
PS C:\Users\radua> Install-Module Pscx

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its InstallationPolicy value by running the Set-PSRepository cmdlet. Are you sure you want to install the modules from 'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): y
PackageManagement\Install-Package : A command with name 'gcb' is already available on this system. This module 'Pscx' may override the existing commands. If you still want to install this module 'Pscx', use -AllowClobber parameter.
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:1661 char:21
+ ...          $null = PackageManagement\Install-Package @PSBoundParameters
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], Exception
    + FullyQualifiedErrorId : CommandAlreadyAvailable,Validate-ModuleCommandAlreadyAvailable,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage

PS C:\Users\radua>
```

```
PS C:\Users\radua> Get-Command Get-Clipboard

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Cmdlet          Get-Clipboard                                      3.1.0.0    Microsoft.PowerShell.Management
```

**Comment by rkeithhill** [comment]
Thanks for reporting this.  I need to get an update out that fixes a number of issues WRT PS 5.0.

**Comment by u0m3** [comment]
As for a quick fix I used
```powershell
Save-Module -Name pscx -Path <your_chosen_path> # I used C:\Users\radua\OneDrive\Documents\WindowsPowerShell\Modules
Import-Module pscx -Prefix pscx
```

**Comment by veleek** [comment]
This issue is a duplicate of #23 which is a little more broad so this could probably just be closed now.

**Comment by rkeithhill** [comment]
Agreed.

**Comment by ilatypov** [comment]
> ```powershell
> Import-Module pscx -Prefix pscx
> ```

I get an error,
> PS C:\Users\a-XXXX> Save-Module -Name pscx -Path D:\Users\XXXX\ps-module-pscx-conflicts-with-gcb
> PS C:\Users\a-XXXX> Import-Module pscx -Prefix pscx
> Import-Module : The specified module 'pscx' was not loaded because no valid module file was found in any module directory.
> At line:1 char:1
> \+ Import-Module pscx -Prefix pscx
> \+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     \+ CategoryInfo          : ResourceUnavailable: (pscx:String) [Import-Module], FileNotFoundException
>     \+ FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand

**Comment by rkeithhill** [comment]
When you save the module to a folder that isn't in your $env:PSModulePath, you need to load it by path rather than name e.g.:
```
Import-Module D:\Users\XXXX\ps-module-pscx-conflicts-with-gcb\pscx.psd1 -Prefix pscx
```

**Comment by ilatypov** [comment]
Thanks, it worked even with just the path to the directory specified in Save-Module.

I figured my local account from `BUILTIN\Administrators` would show its privileges changed, yet a repeated run of the same script would show the privilege unchanged.

`Install-Module pscx -scope CurrentUser
Save-Module -Name pscx -Path d:\util\pscx`

`Import-Module d:\util\pscx -prefix pscx
$p = Get-pscxPrivilege
$p | ft Name, Status -a
$p.Enable('SeTcbPrivilege')
$p.Enable('SeCreateTokenPrivilege')
$p.Enable('SeAssignPrimaryTokenPrivilege')
$p | ft Name, Status -a`

In the end I figured that I am tricked by some other lockdown policy or antivirus, so I stopped my attempts to enable the privileges required for Cygwin `seteuid` work in this custom local account.  I simply reused the `SYSTEM` account to run sshd.
